### ISSUE TITLE
Private message composer needs a 'confirm send' dialog

### ISSUE THREAD

**Issue by baka-kaba** [Awful/Awful.apk#532]
Should match the one in the post composer (without the preview since you can't do that with PMs)

**Comment by Sereri** [comment]
Well technically you _could_ use the same request and just "preview" to the awful thread

**Comment by baka-kaba** [comment]
You mean do a thread preview request? I don't think you can do that without a thread ID, I just tried removing that from the request (deleted it from the form in Chrome Dev Tools and then hit Preview) and it complains that it can't find that thread. Hardcoding a 'safe' open thread's ID would work but it's real hacky... that's what we're gonna do isn't it

**Comment by Sereri** [comment]
Yeah that's what I meant, the Awful thread's id is a save as we're gonna get

Am 28.09.2017 17:28 schrieb "baka-kaba" <notifications@github.com>:

**Comment by baka-kaba** [comment]
Ok I merged in a basic fix for this, i.e. copy and paste the code from the reply version!

I'll leave this open since it really needs reworking, too much duplicated code between the PM reply fragment and the post reply/edit fragment. They're essentially the same thing in a lot of ways. And it's probably worth trying out the PM preview idea too
